src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40 * days;
+  const weekCostOff = 50;
+  const threeDaysCostOff = 20;
+
+  if (days >= 7) {
+    return carCost - weekCostOff;
+  } else if (days >= 3) {
+    return carCost - threeDaysCostOff;
+  } else {
+    return carCost;
+  }
```

Comment:
If you add retun inside if block else redundant

```suggestion
  if (days >= 7) {
    return carCost - weekCostOff;
  }
  
  if (days >= 3) {
    return carCost - threeDaysCostOff;
  }
  
  return carCost;
  
```